wandb/sdk/launch/agent/agent.py

```diff
@@ -579,13 +589,28 @@ def _check_run_finished(self, job_tracker: JobAndRunStatusTracker) -> bool:
         try:
             run = job_tracker.run
             status = run.get_status().state
-            if status in ["stopped", "failed", "finished"]:
+            if status in ["stopped", "failed", "finished", "disconnected"]:
```

Comment:
is disconnected the right word here? potentially preempted?

Reply:
`preempted` sounds good to me. I wasn't totally sold on `disconnected`